wazo_confd/plugins/user_import/creators.py

```diff
@@ -163,6 +164,26 @@ def create(self, fields, tenant_uuid):
         return self.service.create(SIP(tenant_uuid=tenant_uuid, **form))
 
 
+class WebRTCCreator(SipCreator):
+    def create(self, fields, tenant_uuid):
+        #  TODO(pc-m): This will need to get changed to "transport-wss" when PJSIP transport gets merged
+        fields.update(
+            {
+                'options': [
+                    ['transport', 'wss'],
```

Comment:
This line conflicts with the PJSIP transport branch and should be changed to `transport-wss`

Reply:
> This line conflicts with the PJSIP transport branch and should be changed to `transport-wss`

I will wait for the PJSIP transport to be merged before merging then :+1:

Reply:
PJSIP transports is merged. This branch has been rebased.